### ISSUE TITLE
users and their referrals list

### ISSUE THREAD

**Issue by rkhyd** [bradx954/woocommerce-points-and-rewards-referral-extension#2]
Is it possible to have the list of user and their referrals listed?

**Comment by bradx954** [comment]
Yes. The referral information is stored as user meta as follows.

wdm_store_referral: referral id of the referrer.
wdm_store_referral_used: true/false value of whether the referrer has been rewarded for this user.
wdm_store_referral_id: The referral id of the user.

This information can accessed either with the get_user_meta() function. Or directly through the wp_usermeta table.

So to make a table you could use the get_users function and generate a row for each user.

**Comment by bradx954** [comment]
Actually I recently posted another project that might help you with this. https://github.com/bradx954/User-Meta-Table-Editor
